web/src/components/admin/connectors/AutoSyncOptions.tsx

```diff
@@ -0,0 +1,70 @@
+import { TextFormField } from "@/components/admin/connectors/Field";
+import { useFormikContext } from "formik";
+import { ValidAutoSyncSources } from "@/lib/types";
+import { Divider } from "@tremor/react";
+
+// The first key is the connector type, and the second key is the field name
+const autoSyncConfig = {
+  google_drive: {
+    customer_id: {
+      label: "Google Workspace Customer ID",
+      subtext: (
+        <>
+          The unique identifier for your Google Workspace account. To find this,
+          checkout the{" "}
+          <a
+            href="https://support.google.com/cloudidentity/answer/10070793"
+            target="_blank"
+            className="text-link"
```

Comment:
Would move this to a separate `.ts` file, especially in anticipation of this list of connectors growing longer in the future!